src/lib/containers/table/datasets/DatasetItemsEditor.tsx

```diff
@@ -400,7 +400,7 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
 
         <Button
           variant="outline"
-          disabled={selectedIds.size === DATASET_MAX_ITEM_LIMIT}
+          disabled={datasetToUpdate == null}
           onClick={() => setShowEntityFinder(true)}
         >
           Add Items
```

Comment:
Removing the max number of items requirement, and just restricting adding items until the dataset is loaded.

I don't mind removing it here because this button isn't a particularly helpful place to enforce the requirement anyways. The user could still end up getting the error from the server. We should enforce the requirement in the UI with a conditional warning or in the Entity Finder.

Additionally, the limit is changing for Views, so it may change for datasets as well.